test/integration/targets/azure_rm_virtualnetworkpeering/tasks/main.yml

```diff
@@ -0,0 +1,110 @@
+- name: Prepare random number
+  set_fact:
+    vnetname1: "vnet1{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+    vnetname2: "vnet2{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+    peering_name: "peering1{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 100 | random }}"
+
+- name: Create first virtual network
```

Comment:
can we add the test to azure_rm_virtualnetwork test as it's dependent?

Reply:
i checked azure_rm_virtualnetwork test already, it's a long test list, i think it's more efficient to create a new test for vent peering in case any change in peering will need run whole vnet test. In one word, I have same idea before, after looked at existing test, created new one.